public/content/developers/docs/data-and-analytics/block-explorers/index.md

```diff
@@ -25,6 +25,7 @@ You should understand the basic concepts of Ethereum so you can make sense of th
 - [Sirato](https://www.web3labs.com/sirato)
 - [EthVM](https://www.ethvm.com/)
 - [DexGuru Block Explorer](https://ethereum.dex.guru/)
+- [lazy-etherscan](https://github.com/woxjro/lazy-etherscan)
```

Comment:
@woxjro Can just add something here similar to language callouts.

```suggestion
- [lazy-etherscan](https://github.com/woxjro/lazy-etherscan) -_Command-line/terminal UI_
```

As I look at this again, it still feels awkward having this under "Services" since it's more of a tool the user can self-host, and tap into any RPC they'd like. 

I'd suggest we create a new heading here for "Open source tools" perhaps, and place lazy-etherscan and Otterscan in there (and any others I may have missed that work similarly)

```suggestion

## Open source tools {#open-source-tools}

- [Otterscan](https://otterscan.io/)
- [lazy-etherscan](https://github.com/woxjro/lazy-etherscan)
```

(Then remove Otterscan from above list)